src/Adapter/Customer/QueryHandler/SearchCustomersHandler.php

```diff
@@ -37,6 +38,20 @@
  */
 final class SearchCustomersHandler implements SearchCustomersHandlerInterface
 {
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @param Configuration $configuration
+     */
+    public function __construct(
+        Configuration $configuration
+    ) {
+        $this->configuration = $configuration;
+    }
```

Comment:
This is a BC break because you're changing a public method signature, if a module or anything outside of the core was using this class, it won't work anymore after this PR is merged. That's why we need to document it :)

Reply:
Okay, thanks for the explanation!